Makefile

```diff
@@ -81,6 +81,8 @@ OVS_REPO            ?= https://github.com/openstack-k8s-operators/ovs-operator.g
 OVS_BRANCH          ?= main
```

Comment:
would be nice to point to https://github.com/openstack-k8s-operators/ovs-operator/pull/60 in the commit message

i know we don't have depends-on here but it would be useful even for the reviewer